### DISCUSSION TITLE
Allow methods such as set_points_smoothly() to accept iterators as input.

### DISCUSSION THREAD

**Discussion by ubaldot** [ManimCommunity/manim#3380] [Ideas]
Consider the following example:

```
x = [0, 1, 2, 3]
y = [5, 2, 3, 7]
shape = mn.VMobject().set_points_smoothly(list(zip(x,y)))
```

It would be nice if methods such as `set_points_smoothly()` would take iterators as input, e.g. the last line of the above snippet could be `shape = mn.VMobject().set_points_smoothly(zip(x,y))`.

**Comment by MrDiver** [comment]
In Progress
